ddl/db_change_test.go

```diff
@@ -80,9 +80,10 @@ func (s *testStateChangeSuite) TearDownSuite(c *C) {
 func (s *testStateChangeSuite) TestShowCreateTable(c *C) {
 	tk := testkit.NewTestKit(c, s.store)
 	tk.MustExec("use test")
-	tk.MustExec("create table t (id int, index idx (id))")
+	tk.MustExec("create table t (id int)")
 
 	var checkErr error
+	addIndexSeq := 0
```

Comment:
Could you add a comment for this variable?

Reply:
I update the test code.
PTAL @shenli